cursive-core/src/views/dialog.rs

```diff
@@ -349,6 +395,11 @@ impl Dialog {
         self.padding.top = padding;
     }
 
+    /// Gets the top padding in the dialog (under the title).
+    pub fn get_padding_top(&mut self) -> usize {
```

Comment:
Not sure we need all these, could just let the user do `.get_padding().top`.

Reply:
I added these for parity with the setters; same thing with `get_padding_lrtb`. If you want I can remove these.